docs/advanced-orderflow.md

```diff
@@ -0,0 +1,86 @@
+# Advanced Orderflow
+
+This guide walks you through utilizing public trade data for advanced orderflow analysis in Freqtrade.
+
+
+## Getting Started
+
+1. **Enable Public Trades:** in your `config.json` file, set the `use_public_trades` option to true under the `exchange` section.
+```json
+"exchange": {
+   ...
+   "use_public_trades": true,
+}
+```
+2. **Configure Orderflow Processing:** Define your desired settings for orderflow processing within the orderflow section of config.json. Here, you can adjust factors like:
+
+- `scale`: This controls the price bin size for the footprint chart.
+- `stacked_imbalance_range`: Defines the minimum consecutive imbalanced price levels required for consideration.
+- `imbalance_volume`: Filters out imbalances with volume below this threshold.
+- `imbalance_ratio`: Filters out imbalances with a ratio (difference between ask and bid volume) lower than this value.
+
+```json
+"orderflow": {
+    "scale": 0.5, 
+    "stacked_imbalance_range": 3, # needs at least this amount of imblance next to each other
+    "imbalance_volume": 1, # filters out below
+    "imbalance_ratio": 300 # filters out ratio lower than
+  },
```

Comment:
If it's a ratio - isn't 300 a bit excessive? (like - a ratio should go from 0 to 1 (which means 100%)) ?

Reply:
I would like to keep it this way, because I'm cross-referencing this settings with ATAS to compare and check that my stuff works. `300` stands for 300%. should I rename `imbalance_ratio` to `imbalance_percent` to make it more clear?

Reply:
well essentially - that's my point.
it says "imbalance_ratio" - so if it's a ratio - then i assume 1 == 100% (which quite honestly, i'd prefer from a setup perspective, as that'd make it more aligned with the rest of freqtrade settings).